CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
-## Unreleased
+## [2.2.0] - 2020-12-18
 
 ### Added
 
 - Added email suspended node support in sample apps
+- Added support for `TextOutputCallback` `messageType 4` behind a configuration option that is disabled by default
+- Support for TypeScript 4.0
 
```

Comment:
Do you think it worth mentioning the improvements related to the combinations of nodes within a page node? (SDKS-811)

Reply:
It was already mentioned under the Fixed header: https://github.com/ForgeRock/forgerock-javascript-sdk-ui/pull/40/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR13

Reply:
I must have been blind... not sure how I missed it...